veloroutes_voies_vertes/processing/covadis/export_covadis.py

```diff
@@ -113,18 +113,17 @@ def initAlgorithm(self, config):
         # Chemin du dossier de destination
         outparam = QgsProcessingParameterFolderDestination(
             self.PROJECTS_FOLDER,
-            tr("Chemin de destination"),
-            '',
-            False,
-            False
+            description=tr("Chemin de destination"),
+            defaultValue='',
+            optional=True
         )
         outparam.tooltip_3liz = 'Chemin de destination pour enregistrer les exports Shapefile'
         self.addParameter(outparam)
 
         param = QgsProcessingParameterBoolean(
             self.CHARGER,
             tr("Charger la couche correspondante dans le projet"),
-            defaultValue=False,
+            defaultValue='',
```

Comment:
```suggestion
            defaultValue=False,
```
Pas besoin de modifier cette valeur